spec/forms/candidate_interface/becoming_a_teacher_form_spec.rb

```diff
@@ -25,18 +27,54 @@
   end
 
   describe '#save' do
-    it 'returns false if not valid' do
-      becoming_a_teacher = described_class.new
+    context 'transaction succeeds' do
+      it 'updates the provided ApplicationForm' do
```

Comment:
```suggestion
      it 'updates the provided ApplicationForm', aggregate_failures: true do
```
I would either split these expectations up or use `aggregate_failures` since it can be ambiguous which expectation fails